thrift/lib/cpp2/protocol/test/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+enable_testing()
```

Comment:
This should only be called in source directory root per https://cmake.org/cmake/help/latest/command/enable_testing.html.